types/budgets-json.d.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+    module LH {
+        module BudgetsJSON {
+            /** More info: https://github.com/GoogleChrome/lighthouse/issues/6053#issuecomment-428385930 */
+            export interface Budget {
+                resourceCounts?: Array<ResourceBudget>;
+                resourceSizes?: Array<ResourceBudget>;
+                timings?: Array<TimingBudget>;
+            }
+
+            export interface ResourceBudget {
+                resourceType: string;
```

Comment:
could start the better types here or in the next PR

@brendankenny might have more thoughts